tests/test_docker.sh

```diff
@@ -7,10 +7,14 @@
 set -ex
 
 # run the model service's tests
-docker-compose -f local.yml run prediction pytest -rsx
+# Coverage should ignore pip packets, files including tests and pytest
+docker-compose -f local.yml run prediction coverage run --branch --omit=/usr/local/lib/python3.6/dist-packages/*,src/tests/*,/usr/local/bin/pytest /usr/local/bin/pytest -rsx
+docker-compose -f local.yml run prediction coverage report
 
 # run the backend API tests
-docker-compose -f local.yml run interface python manage.py test
+# Coverage should ignore pip packets, files including migrations and tests
+docker-compose -f local.yml run interface coverage run --branch --omit=/usr/local/lib/python3.6/dist-packages/*,**/migrations/*.py,**/test*.py manage.py test
```

Comment:
Can we make the python version not be hardcoded here? perhaps with `*/dist-packages/*` ?

Reply:
Good point. Thanks, I adapted it.